java/system-test/system-test-suite/src/main/java/sleeper/systemtest/suite/dsl/SystemTestCompaction.java

```diff
@@ -47,7 +47,6 @@ public SystemTestCompaction forceCreateJobs() {
     }
 
     public SystemTestCompaction splitAndCompactFiles() throws InterruptedException {
-        createJobs().invokeTasks(1).waitForJobs();
         forceCreateJobs().invokeTasks(1).waitForJobs(
                 PollWithRetries.intervalAndPollingTimeout(Duration.ofSeconds(5), Duration.ofMinutes(30)));
```

Comment:
I think we want to test the standard compaction process rather than forcing compaction. Does this pass with createJobs instead of forceCreateJobs?

Reply:
When I ran tests with this call, the `CreateJobs` lambda didnt create any jobs, because the batch size was not met. Compactions for both system tests that use this code (`MultipleTablesIT` and `PartitionSplittingIT`) create compactions with one file in them.

The CompactionPerformanceIT test still uses `createJobs`, although I've yet to run this performance test

Reply:
I've created an issue for creating functional system tests that use the lambda instead of the `CreateJobs` class directly: https://github.com/gchq/sleeper/issues/1710